### PR TITLE
Add yes flag

### PR DESCRIPTION
**Pull request by UnleashedMind**
- Add `--yes` flag for the start and init command. 
When `--yes` flag specified, the command will execute without console prompt and use default values for all the prompt questions. 
- fixed the console logged url for new user setup.

### CODE REVIEW

File: lib/backend-create.js [hunk 1]
```diff
@@ -27,52 +27,60 @@ const nameManager = require('./utils/awsmobilejs-name-manager.js')
...
-                        console.log(chalk.red('something went wrong'))
-                    }
-                })
-            } 
-        })
+                    backendInfoManager.syncCurrentBackendInfo(projectInfo, backendDetails, awsDetails, options.syncToDevFlag, function(){
+                        if(callback){
+                            callback()
+                        }
+                    })
```
Comment (elorzafe) [inline lib/backend-create.js]: There is no message if backendWaitLogic fails

File: lib/backend-create.js [hunk 2]
```diff
@@ -27,52 +27,60 @@ const nameManager = require('./utils/awsmobilejs-name-manager.js')
...
+                }
+            })
+        }else{
+            console.log(chalk.red('something went wrong'))
```
Comment (elorzafe) [inline lib/backend-create.js]: If that happen should be a more verbose message.

File: lib/command-init.js [hunk 3]
```diff
@@ -20,9 +20,16 @@ const setupBackend = require('./init-steps/s5-setup-backend.js')
 const onSuccess = require('./init-steps/s60-on-success.js')
 const onFailure = require('./init-steps/s61-on-failure.js')
 
-function init(mobileProjectID){
+function init(mobileProjectID, yesFlag){
 
-    return analyzeProject.run(mobileProjectID)
+    let projectPath = process.cwd()
+    let initInfo = {
+        projectPath: projectPath,
```
Comment (elorzafe) [inline lib/command-init.js]: you can use 
{
  projectPath, 
  yesFlag, 
  mobileProjectID
}

File: lib/init-steps/s1-analyze-project.js [hunk 4]
```diff
@@ -36,24 +36,19 @@ const projectStage = [
...
-                mobileProjectID: mobileProjectID,
-                backupAWSMobileJSDirPath: undefined,
-                projectInfo: getProjectInfo(projectPath),
-                projectConfig: getProjectConfig(projectPath),
-                backendProject: getBackendProject(projectPath),
-                packageJson: getPackageJson(projectPath)
-            }
+        if(fs.existsSync(initInfo.projectPath)){
+            initInfo.backupAWSMobileJSDirPath = undefined
+            initInfo.projectInfo = getProjectInfo(initInfo.projectPath)
```
Comment (elorzafe) [inline lib/init-steps/s1-analyze-project.js]: What happen is getProjectInfo fails?

Comment (UnleashedMind) [reply lib/init-steps/s1-analyze-project.js]: getProjectInfo will examine the project-info.json file, if it does not exist, or if the Json parser fails, or the init records is not present in the file, the getProjectInfo function will return undefined. so the initInfo.projectInfo = undefined. that means, the current project that is being init'ed does not contain valid project info, and the following steps will acting accordingly.

File: lib/backend-create.js (hunk 1 above)
Comment (UnleashedMind) [reply lib/backend-create.js]: the backend-wait-loic.js file has the implementation, when the wait log has failed, proper error messages are printed out on the console. 
https://github.com/aws/awsmobile-cli/blob/39bf9c7c718a1090ae99afee271bee55cd5afed7/lib/backend-wait-logic.js#L65

File: lib/backend-create.js (hunk 2 above)
Comment (UnleashedMind) [reply lib/backend-create.js]: I put it there just to be safe, it shouldn't happen, if it happens, it means the call returns with success but the expected data is not there, so the cli prints out something just so the user has some feedback.

**Pull request merged**